examples/excelformer.py

```diff
@@ -20,17 +20,19 @@
 from torch_frame.data.loader import DataLoader
 from torch_frame.datasets.yandex import Yandex
 from torch_frame.nn import ExcelFormer
+from torch_frame.nn.utils.loss import cross_entropy_with_logits
 from torch_frame.transforms import CatToNumTransform, MutualInformationSort
 
 parser = argparse.ArgumentParser()
-parser.add_argument('--dataset', type=str, default='higgs_small')
+parser.add_argument('--dataset', type=str, default='california_housing')
+parser.add_argument('--mixup', type=str, default='feature',
+                    choices=['none', 'ordinary', 'feature', 'hidden'])
```

Comment:
What is ordinary? Shouldn't it be none, feature, or hidden?

Reply:
As replied above, the `ordinary` is the previous mixup version implemented in this project, thus I left that option in the codes, while in our paper, the proposed `feature` & `hidden` are always better than `ordinary`. As suggested, I have removed `ordinary` and only keep the `none` & `feature` & `hidden`.